docs/en/modules/admin/pages/spaces/processes/types.adoc

```diff
@@ -0,0 +1,38 @@
+= Process types
+
+In some cases it is necessary to organize participatory processes by more than just xref:spaces/processes/groups.adoc[Process Groups]. This is where Process Types can be useful.
+
+Process Types work similarly to xref:assemblies:assemblies.adoc#_assemblies_types[Assembly types].
+
+== New process type form
+
+For creating a new process type:
+
+. Sign in as administrator
+. Go to the Processes section in the sidebar
+. Click on the Process types link in the sidebar
+. Click on the "New process type" button
+. Fill in the form
+
+image:spaces/processes/process-types-admin-backend.png[A screenshot of the back-end showing the button to create a new process type,title="Create a new process type in the back-end"]
+
+.New participatory process type form: General information
+|===
+|Field |Type |Description
+
+|Title
+|Required
+|Title of the participatory process type. For instance, "Consultative".
+
```

Comment:
There's a missing closing table tag here. Without this the [HTML is broken](https://deploy-preview-138--decidim-documentation.netlify.app/en/develop/admin/spaces/processes/types):

![Screenshot of the process types pages from netlify preview deployment](https://github.com/decidim/documentation/assets/717367/eaabe56e-3db3-4a12-a8b9-f48ee4263d93)

```suggestion

|===

```